Parse/src/main/java/com/parse/NotificationCompat.java

```diff
@@ -124,6 +128,7 @@ public Notification build(Builder b) {
     Bitmap mLargeIcon;
     int mPriority;
     Style mStyle;
+    NotificationChannel mNotificationChannel;
```

Comment:
Sorry man, I just noticed this... I am not sure if this might throw at runtime on older APIs, even if not used. Could we just store `String mNotificationChannelId` here?